package.json

```diff
@@ -30,7 +30,7 @@
     "dist:mac": "electron-builder --mac",
     "dist:linux": "electron-builder --linux --x64 --publish onTag",
     "pack": "yarn && yarn build && yarn clean:dist && yarn dist",
-    "pack:win": "cmd /c yarn && yarn install:electron:dependencies && yarn clean:build && yarn copy:files && yarn build:prod && yarn build:electron && yarn clean:dist && yarn dist:win",
+    "pack:win": "cmd /c yarn && yarn install:electron:dependencies && yarn copy:files && yarn build:prod && yarn build:electron && yarn dist:win && yarn clean:dist:win",
```

Comment:
The change here now prevents building an unsigned app on Windows

Reply:
if you use windows powershell it should work?

Reply:
It doesn't work with Windows Powershell too. The issue is that the app signing CLI, `smtcl` isn't installed on my computer and I shouldn't need to install it if I'm building an unsigned version of the app.

Reply:
ah ok.. I think now that we have this job configured.. we can easily create tag and delete the tag later for testing on windows..